protostar/commands/test/test_collector.py

```diff
@@ -16,17 +18,35 @@
 from protostar.protostar_exception import ProtostarException
 from protostar.utils.starknet_compilation import StarknetCompiler
 
+TestSuiteGlob = str
+TestSuiteFilepath = str
+TestCaseGlob = str
+TestSuiteAndCaseGlob = str
+"""e.g. `tests/**/::test_*`"""
+TestCasesMapping = Dict[TestSuiteFilepath, Set[TestCaseGlob]]
+
+
+@dataclass
+class TargetTestCasesInfo:
```

Comment:
This is more a Suite Info, also target is a weird word. Maybe?
```suggestion
class CollectedTestSuite:
```

Reply:
Maybe `TargetTestCasesGlobs`?

Reply:
An instance of `TargetTestCasesGlobs` should use snake case, and the snake version of this class would suggest a list or set of globs.

`CollectedTestSuite` could suggest that this class inherits from `TestSuite`.

I'll change it to `TestSuiteInfo`.